dcos-log/mesos/files/reader/read.go

```diff
@@ -349,9 +369,68 @@ start:
 	return strings.NewReader(rm.formatFn(*line, rm)).Read(b)
 }
 
-// Sandbox returns a path to mesos sandbox.
-func (rm ReadManager) Sandbox() string {
-	return rm.sandboxPath
+// SandboxFile represents a file object located in mesos sandbox.
+type SandboxFile struct {
+	GID   string `json:"gid"`
+	Mode  string `json:"mode"`
+	MTime mTime  `json:"mtime"`
+	NLink uint   `json:"nlink"`
+	Path  string `json:"path"`
+	Size  uint64 `json:"size"`
+	UID   string `json:"uid"`
+}
+
+type mTime uint64
+
+// mtime field has a weird format with trailing .0
+// in order to unmarshal this value into json, we need to cut off the suffix .0 if it exists
+func (mt *mTime) UnmarshalJSON(data []byte) error {
+	sanitizedData := bytes.TrimSuffix(data, []byte(".0"))
```

Comment:
do we know that the suffix will always be `.0` ? if not, it may make sense to trim everything after, and including the `.` rune.

Reply:
yeah, it's kind of weird response, but it will always be `.0`